site/index.html

```diff
@@ -0,0 +1,146 @@
+        He holds two world records for becoming the fastest bowler to take 100 wickets in T20 and ODI formats.
+        His skill shines in international matches and various T20 leagues worldwide.
+      </p>
+      <p>
+        Off the field, Sandeep is also known for his passion for music.
+        Though he faced serious legal challenges, his conviction was overturned.
+        He remains a key figure in Nepalese cricket, inspiring many young players.
+        His journey reflects talent, resilience, and determination to succeed.
+      </p>
+
+      <!-- Responsive image of myself -->
+      <section id="thumbnail-and-alternative-sizes">
+        <a href="images/myself.png">
+          <picture>
+            <source media="(max-width: 500px)" srcset="images/thumbnail-150.png" />
+            <source media="(min-width: 501px)" srcset="images/thumbnail-300.png" />
+            <img src="images/thumbnail-300.png" alt="Photo of myself" />
+          </picture>
+        </a>
+      </section>
+    </section>
+
+    <!-- Favourite Things Section -->
+    <section id="my-favourite-things">
+      <h2>My Favourite Things</h2>
+
+      <!-- Linked image wrapped in figure with caption -->
+      <figure>
+        <a href="https://en.wikipedia.org/wiki/Sandeep_Lamichhane" target="_blank" rel="noopener noreferrer">
+          <img src="images/logo.png" alt="Photo of Sandeep Lamichhane, Nepalese cricketer" style="width: 150px; height: auto;" />
+        </a>
+        <figcaption>Sandeep Lamichhane – Nepal's cricket sensation</figcaption>
+      </figure>
+
+      <!-- Ordered list of favourite things -->
+      <ol>
+        <li>Travelling</li>
+        <li>Playing Cricket and Football</li>
+        <li>Singing and Listening Songs</li>
+      </ol>
+    </section>
+
+    <!-- Haiku -->
+    <section id="haiku">
+      <h3>Haiku</h3>
+      <p>
+        Practice every day<br />
+        Chase dreams like runs on the pitch<br />
+        Greatness will follow<br />
+        <br />
+        <span class="author">– Sachin Tendulkar</span>
+      </p>
+    </section>
+
+    <!-- Embedding External Content -->
+    <section id="embedding-external-content">
+      <h3>Public Domain Audiobook</h3>
+      <audio controls>
+        <source src="https://dn721507.ca.archive.org/0/items/booksandreading_2505_librivox/booksandreading_09_porter_128kb.mp3" type="audio/mpeg" />
+        <p>Your browser doesn't support audio.
+          <a href="https://dn721507.ca.archive.org/0/items/booksandreading_2505_librivox/booksandreading_09_porter_128kb.mp3">Download the audiobook</a>.
+        </p>
+      </audio>
+
+      <h3>YouTube Video</h3>
+      <iframe width="560" height="315"
+              src="https://www.youtube.com/embed/j61j9X4xCnA"
+              title="Visit Nepal - Lifetime Experience"
+
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+              allowfullscreen>
+      </iframe>
```

Comment:
> 10. `Lab 3 > Embedded external content > embedded video via iframe`
Missing referrer policy.
```suggestion
      <iframe width="560" height="315"
              src="https://www.youtube.com/embed/j61j9X4xCnA"
              title="Visit Nepal - Lifetime Experience"
              referrerpolicy="strict-origin-when-cross-origin"
              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
              allowfullscreen>
      </iframe>
```